jest.config.js

```diff
@@ -38,6 +39,8 @@ module.exports = {
     "\\.(jpg|ico|jpeg|png|gif|eot|otf|webp|svg|ttf|woff|woff2|mp4|webm|wav|mp3|m4a|aac|oga)$":
       "<rootDir>/config/jest/fileTransform.js",
     "\\.(css|less)$": "<rootDir>/config/jest/fileTransform.js",
+    "^lodash-es$": "lodash",
```

Comment:
We should probably update the tests so that we don't have this legacy/tech-debt mapping going on.

Reply:
Fixed. Tell Jest not to ignore lodash-es when transforming
<img width="739" alt="Screenshot 2024-07-31 at 15 39 50" src="https://github.com/user-attachments/assets/54367dfd-b103-4647-b2e0-60db8bad5660">